eks/modules/nodes/variables.tf

```diff
@@ -0,0 +1,35 @@
+variable "name" {
+  description = "Prefix for resource names and tags"
+  type        = string
+}
+
+variable "worker_groups" {
+  description = "Worker groups definition list"
+  type        = list(object({
+    name                 = string
+    instance_type        = string
+    asg_desired_capacity = number
+    asg_min_size         = number
+    asg_max_size         = number
+  }))
+}
+
+variable "subnet_ids" {
+  description = "Subnet ids to join to"
+  type = list(string)
+}
+
+variable "disk_size" {
+  description = "Disk size"
+  type        = number
+}
```

Comment:
Maybe we want to add some validations here?

Reply:
Variables inside nested modules are reusing variables from root .tftvars so validation here is unnecessary.